charts/flagsmith/values.yaml

```diff
@@ -112,6 +112,46 @@ frontend:
     successThreshold: 1
     timeoutSeconds: 10
 
+# See https://docs.flagsmith.com/deployment/task-processor
+taskProcessor:
+  enabled: false
+  replicacount: 1
+  sleepIntervalMs: null
+  numThreads: null
+  gracePeriodMs: null
+
+  livenessProbe:
+    failureThreshold: 5
+    initialDelaySeconds: 5
+    periodSeconds: 10
+    successThreshold: 1
+    timeoutSeconds: 2
+  readinessProbe:
+    failureThreshold: 10
+    initialDelaySeconds: 1
+    periodSeconds: 10
+    successThreshold: 1
+    timeoutSeconds: 2
+
+  podAnnotations: {}
+  resources: {}
+  # limits:
+  #   cpu: 500m
+  #   memory: 500Mi
+  # requests:
+  #   cpu: 300m
+  #   memory: 300Mi
+  podLabels: {}
+  nodeSelector: {}
+  tolerations: []
+  affinity: {}
+  podSecurityContext: {}
+  defaultPodSecurityContext:
+    enabled: true
+    # runAsNonRoot: true  # TODO: enable this, conditional on tag semver
```

Comment:
I appreciate this is also in other places in this file but I'm not entirely sure I understand what this is waiting on? What is meant be 'tag semver' ?

Reply:
There will - presumably - be some application version that means that setting this to `true` becomes the right thing to do, but before which setting this would break things.

Suppose it was application version 1.2.3 that introduced this change, then would probably want the chart behaviour as follows:
- if `runAsNonRoot` not set (or set to null) in values, then
    - if image tag >= 1.2.3, do set `runAsNonRoot: true` for the pods, as this is known to be safe
    - else do not set `runAsNonRoot`
- if `runAsNonRoot` set in values to `true` or `false`, then set that for the pods, as that is what the user has asked for.

That might be important enough to get ticketed, but maybe not.